test/h3core.spec.js

```diff
@@ -145,6 +145,32 @@ test('h3GetResolution', assert => {
     assert.end();
 });
 
+test('h3GetResolution - integers', assert => {
+    // Same as in h3GetResolution above
+    const indexes = [
+        [4294967295, 134389759],
```

Comment:
Not required here, but I think we should probably expose `h3IndexToSplitLong` as a public function and use it here rather than hard-coding the input data. If this test fails, it'll be very hard for a future engineer to determine whether it's the fault of the code or the input.